redis_json/src/commands.rs

```diff
@@ -579,7 +585,7 @@ pub fn json_del<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>) -
 }
 
 ///
-/// JSON.MGET <key> [key ...] <path>
+/// JSON.MGET <key> [key ...] [FORMAT STRING|JSON|EXPAND] <path>
```

Comment:
>This API could be very confusing, for example what if the keys I want to work on are FORMAT STRING, this will obligate 
> me to specify the FORMAT as mandatory.
Yes @LiorKogan 's decision....

Reply:
:face_with_head_bandage:

Reply:
And what about the command definition which is wrong?

Reply:
> ["json.mget", json_command!(json_mget), "readonly", 1,1,1],

Yes, I'm not trying to protect from cross on the Redis level.
This means that you must catch cross slot errors in the command implementation itself.

Reply:
@gkorland It was decided for `JSON.MGET` to
* Avoid adding the `FORMAT` arg
* Use default `EXPAND` in resp3 and default `STRING` in resp2
* Not to fix the command key position (from 1,1,1 to 1,-2,1) to avoid breaking change in resp2
  * @LiorKogan will document this existing problem
* Need a new command that follows the redis supported key positions